### DISCUSSION TITLE
Who designed the Prisma.io website?

### DISCUSSION THREAD

**Discussion by holdingsllc** [prisma/prisma#2583] [Q&A] [answered]
Because I want to hire this person. chris@cdbholdings.com or cdbarlow@gmail.com

**Comment by nikolasburk** [comment, marked as answer]
Hey @holdingsllc 👋 we have an internal design team which mainly consists of @kuldar and @hrescak at the moment! I'm afraid they're not for hire right now though 🙈

**Reply by holdingsllc** [reply]
Thanks for the insight. Do you know when @kuldar or @hrescak will be available?

**Reply by kuldar** [reply]
Haha, thanks for the interest @holdingsllc but currently there's plenty enough to work on in Prisma :)
